hosts/groups/meta/common.nix

```diff
@@ -0,0 +1,76 @@
+{config, lib, pkgs, ...}:
```

Comment:
This is still incompatible with `ssh-for-each`, file names in `hosts/` should only contain FQDNs

`/hosts/` is not for meant for this, please move it to `/nixosConfigurations` or `/nixosModules` or add it as a function in `/lib`